.github/workflows/ci.yml

```diff
@@ -0,0 +1,31 @@
+name: Continuous Integration
+on:
+  - pull_request
+  - push
+
+jobs:
+
+  test:
+    name:    Test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Build Common
+        uses: actions/checkout@v2
+        with:
+          path:       build-common
+          repository: paketo-buildpacks/build-common
+      - name: Checkout Source
+        uses: actions/checkout@v2
+        with:
+          path: source
+      - name: Setup Go environment
+        uses: actions/setup-go@v2.0.3
+      - name: Restore Cache
```

Comment:
in my testing a long time ago, using a cache for go mod pkg wasn't a perf gain. Maybe it's gotten better since then

Reply:
After discussion, no change.